### PR TITLE
Added inbox page and fixed hover effect in navigation items

### PR DESCRIPTION
**Pull request by krishnkant-singh**

### CODE REVIEW

File: src/components/navigation/Navigation.tsx [hunk 1]
```diff
@@ -7,33 +8,69 @@ import {
...
   InboxIcon,
   BoltIcon,
+  BellIcon,
+  ListBulletIcon
 } from "@heroicons/react/24/outline";
 
+
+
 export const Navigation = () => {
+  const [activeIndex, setActiveIndex] = useState(-1);
```
Comment (bryanlundberg) [inline src/components/navigation/Navigation.tsx:18]: This is not going to work for a navbar. I suggest to verify the path.

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
Hi @krishnkant-singh could you refactor the logic for detecting the navbar hover status. Also include the real navigation on click. Please 👋

**Comment by krishnkant-singh** [comment]
hi @bryanlundberg i wanna know if active status represents hover thing or the page you are on

**Comment by bryanlundberg** [comment]
> hi @bryanlundberg i wanna know if active status represents hover thing or the page you are on

It's both, represents the style for hover and current active.

**Pull request closed without merging**